include/mbedtls/pk.h

```diff
@@ -476,16 +476,15 @@ int mbedtls_pk_can_do_ext(const mbedtls_pk_context *ctx, psa_algorithm_t alg,
  *                    based on the #mbedtls_pk_type_t type of \p pk,
  *                    the chosen \p usage and other factors:
  *                      - #MBEDTLS_PK_RSA whose underlying
- *                        #mbedtls_rsa_context has the padding mode
- *                        #MBEDTLS_RSA_PKCS_V15:
+ *                        context uses the PKCS#1 v1.5 padding mode:
```

Comment:
Minor nit on commit structure (no need to rewrite history now, just a though for future PR): as a reviewer, I would prefer for a commit to contain either manual changes (such as this one) or "mechanical" changes (such as substituting `PSA_ERROR_INVALID_SIGNATURE` etc below), rather than a mixture of both. No biggie, just a thought for next time.